### PR TITLE
Remove mining db stats - replaced by runtime state variable

### PR DESCRIPTION
**Pull request by nymkappa**
Fixes https://github.com/mempool/mempool/issues/3114

There is no simple way to test this aside waiting for one week to see if the network hashrate and mining pool weekly hashrate indexing is properly triggered. However the code change is trivial so I don't think testing it over a full week is necessary.

Basically we replaced a database variable with a local one. Since we were always resetting those db variable to 0 upon restart, there was no point in having them in the database anymore, in memory variable is good enough.

I've added the following new debug logs to track indexing attempt.
```
Feb 25 11:29:45 [42836] DEBUG: [Mining] Network hashrate history indexing is up to date, nothing to do
Feb 25 11:29:45 [42836] DEBUG: [Mining] Pool hashrate history indexing is up to date, nothing to do
```

### CODE REVIEW

File: backend/src/api/database-migration.ts [hunk 1]
```diff
@@ -632,6 +637,11 @@ class DatabaseMigration {
       queries.push(`INSERT INTO state(name, number, string) VALUES ('last_weekly_hashrates_indexing', 0, NULL)`);
     }
 
+    if (version < 55) {
```
Comment (softsimon) [inline backend/src/api/database-migration.ts:640]: This did not execute. Wrong version number here?

Comment (nymkappa) [reply backend/src/api/database-migration.ts]: Yep, forgot this one while rebasing. Fixed in https://github.com/mempool/mempool/pull/3122/commits/e0c3c732d17d4c05831245f7f07c85def8a3d19e.

**Review by softsimon — APPROVED** [review]
tested ACK @ e0c3c732

**Review by wiz — APPROVED** [review]
Partially Tested ACK @ 1192d4f
```
Mar  4 07:52:31 [31241] NOTICE: Starting Mempool Server... (1192d4f)
Mar  4 07:52:31 [31241] INFO: Database connection established.
Mar  4 07:52:31 [31241] DEBUG: MIGRATIONS: Running migrations
Mar  4 07:52:31 [31241] DEBUG: MIGRATIONS: Database engine version '10.5.17-MariaDB'
Mar  4 07:52:31 [31241] DEBUG: MIGRATIONS: Current state.schema_version 57
Mar  4 07:52:31 [31241] DEBUG: MIGRATIONS: Latest DatabaseMigration.version is 58
Mar  4 07:52:31 [31241] DEBUG: MIGRATIONS: Execute query:
UPDATE state SET number = 58 WHERE name = 'schema_version';
Mar  4 07:52:31 [31241] NOTICE: MIGRATIONS: Upgrading database schema version number to 58
Mar  4 07:52:31 [31241] DEBUG: MIGRATIONS: Execute query:
START TRANSACTION;
Mar  4 07:52:31 [31241] DEBUG: MIGRATIONS: Execute query:
DELETE FROM state WHERE name = 'last_hashrates_indexing'
Mar  4 07:52:31 [31241] DEBUG: MIGRATIONS: Execute query:
DELETE FROM state WHERE name = 'last_weekly_hashrates_indexing'
Mar  4 07:52:31 [31241] DEBUG: MIGRATIONS: Execute query:
UPDATE state SET number = 58 WHERE name = 'schema_version';
Mar  4 07:52:31 [31241] DEBUG: MIGRATIONS: Execute query:
COMMIT;
Mar  4 07:52:31 [31241] NOTICE: MIGRATIONS: OK. Database schema have been migrated from version 57 to 58 (latest version)
Mar  4 07:52:32 [31241] DEBUG: pools-v2.json sha | Current: 71cd80eae5ce6de28f227ddec28bef18cc62b62d | Github: 71cd80eae5ce6de28f227ddec28bef18cc62b62d
Mar  4 07:52:32 [31241] INFO: Restoring mempool and blocks data from disk cache
Mar  4 07:52:34 [31241] INFO: Starting statistics service
Mar  4 07:52:34 [31241] DEBUG: [Mining] Inserted 0 MtGox USD weekly price history into db
Mar  4 07:52:35 [31241] DEBUG: Updated orphaned blocks cache. Found 2 orphaned blocks
Mar  4 07:52:35 [31241] NOTICE: Mempool Server is running on port 8999
```

**Pull request merged**